ReferencePluginA/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="PluginInterfaces" publicKeyToken="583bede96b7c94a3" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-0.21.0.0" newVersion="0.21.0.0" />
```

Comment:
I think Tim pointed out that the app.config file here is not needed, since Paratext's is the one that will be used. I've deleted the ones for our other demo plugins.